concordium-consensus/src/Concordium/GlobalState/Persistent/BlobStore.hs

```diff
@@ -329,6 +329,12 @@ instance HasBlobStore BlobStore where
   blobLoadCallback = bscLoadCallback
   blobStoreCallback = bscStoreCallback
 
+-- |An auxiliary constraint needed by all functions that migrate state from one
+-- blob store to another. The intended reading of this is that @m@ and @t@
+-- support migration from the context @m@ to the context @t m@. The context in
+-- this case is essentially access to a block state database.
+type SupportMigration m t = (MonadBlobStore m, MonadTrans t, MonadBlobStore (t m))
```

Comment:
I had that first. But then I renamed it to the plural since both `m` and `t` together support migration from `m` to `t m`.

So I don't want to rename it back.

Reply:
I think (at least most of) the other constraints have a main `m` with some auxiliary context perhaps, so this constraint is a bit different. In any way, that was my reasoning.